app/models/photo_like.rb

```diff
@@ -0,0 +1,6 @@
+class PhotoLike < ActiveRecord::Base
+  belongs_to :photo
+  belongs_to :user
+
+  validates :user_id, :uniqueness => { :scope => :photo_id }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)